hadoop-cloud-storage-project/hadoop-huaweicloud/src/main/java/org/apache/hadoop/fs/obs/OBSListing.java

```diff
@@ -453,7 +453,7 @@ private boolean buildNextStatusBatch(final ObjectListing objects) {
       }
 
       // prefixes: always directories
-      for (ObsObject prefix : objects.getExtenedCommonPrefixes()) {
+      for (ObsObject prefix : objects.getExtendCommonPrefixes()) {
```

Comment:
the original method is deprecated, this change migrated it to the suggested alternative